core/diagnostics/DiagnosticScenarios/readme.md

```diff
@@ -0,0 +1,32 @@
+# DiagnosticScenarios sample debug target
+
```

Comment:
We should make this downloadable from Samples Browser (<https://docs.microsoft.com/samples/browse>). That requires that we add metadata to the top of the readme:
```
---
languages:
   - csharp
 products:
   - dotnet-core
page_type: sample
name: "Diagnostic Scenarios"
urlFragment: "diagnostic-scenarios"
description: "A .NET Core sample with methods that trigger undesirable behaviors to diagnose."
--- 
```